examples/tdec/simple_tdec_local.py

```diff
@@ -0,0 +1,48 @@
+from nucypher.characters.chaotic import NiceGuyEddie as _Enrico
+from nucypher.characters.chaotic import ThisBobAlwaysDecrypts
+from nucypher.policy.conditions.lingo import ConditionLingo
+
+plaintext = b"paz al amanecer"
+THIS_IS_NOT_A_TRINKET = 55
```

Comment:
I know this is a critique - but perhaps include a comment?

```suggestion
THIS_IS_NOT_A_TRINKET = 55  # sometimes called "public key"
```